src/test/integration/integrationtests.py

```diff
@@ -33,31 +49,33 @@ def test_person_by_ids(self):
 
         # identifiers and return ids should matched
         person_data = osu_id_res.json()['data'][0]
-        ids_list = [
-            (person_data['id'], osu_id),
-            (person_data['attributes']['username'], onid),
-            (person_data['attributes']['osuUID'], osuuid)
-        ]
+        ids_list = [(person_data['id'],
+                     osu_id), (person_data['attributes']['username'], onid),
+                    (person_data['attributes']['osuUID'], osuuid)]
```

Comment:
Regarding all the formatting comments, I ran the code through google's [yapf](https://github.com/google/yapf) with the default settings, which formats to pep8. I confirmed it conforms to pep8 by running this tool: https://github.com/PyCQA/pycodestyle

I want to get out of the business of us manually formatting our code. For example, go has [gofmt](https://golang.org/cmd/gofmt/) to help with this, and I figured yapf could do the similar task for python.

I'm going to leave all these formatting comments as-is right now since the code conforms to pep8. Perhaps in the future we can talk about how we want to agree to use something like yapf so the format is something we can agree on.

Reply:
I'm not opposed to using yapf or similar tools to format the code, but I found that sometimes these tools format the code in a kind of weird way, for example, breaking the lines even it is not really necessary. Similarly, although we also use codenarc to auto-checked the code style for our Groovy codes, we still always be very careful of the coding style when we reviewing the code. The reason behind that is because I think the readability is also very important. In my opinion, it might directly influence the difficulty of understanding.
However, I also understand that coding style could be various depends on persons. That is probably why I think we should discuss the styling stuff whenever doing the code review. We don't really want to wait until the code becomes too large. What do you think?

Reply:
Yeah, that's fair. I'll make a few changes. I think we can agree that ideally, we wouldn't have to manually format code at all. I'm starting to see why Andrew liked go so much. Among other reasons, gofmt seems to be one of the best formatters out there. I agree yapf did do some questionable things.